src/task_system/task_system_server/Validators/PermissionValidator.cs

```diff
@@ -8,8 +8,16 @@ public bool ValidatePermissions(string permissionName, string sessionKey)
         {
             var permData = _permClient.GetSessionData(sessionKey).GetAwaiter().GetResult();
             var permissionList = permData?.Data?.SessionData?.SessionPermissionList ?? [];
-            Console.WriteLine("permissionList:" + permissionList + "| permissionName:" + permissionName);
+
             return permissionList.Contains(permissionName);
         }
+
+        public bool ValidateAccountId(int accountId, string sessionKey)
+        {
+            var permData = _permClient.GetSessionData(sessionKey).GetAwaiter().GetResult();
+
+            Console.WriteLine(permData?.Data?.SessionData?.AccountId + " " + accountId);
```

Comment:
## Non-blocking

Might be better to indicate the context for this print statement eg "Validator: AccountId {accountId} is associated to session key {key}: {true/false}"